### PR TITLE
Fixes 328: 404 page added

### LINKED ISSUE #328: add 404 error page
### This is a
- [ ] Bug report
- [x] Feature request

### Current behaviour

Stack trace is showing
### Expected behaviour

add a bootstrap 404 page

### Steps to reproduce
1.
2.
3.

### PR DESCRIPTION
**Pull request by sch00lb0y**
## Description

I have added 404 page 
## Related Issue

close #328 
## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -93,9 +93,8 @@ io.on('connection', function(socket){
 
 // catch 404 and forward to error handler
 app.use(function(req, res, next) {
-  var err = new Error("Not Found");
-  err.status = 404;
-  next(err);
+  res.status(404)
```
Comment (imujjwal96) [inline app.js]: semicolon

**Comment by imujjwal96** [comment]
@sch00lb0y Can you show us a sample deployment?

**Comment by sch00lb0y** [comment]
@imujjwal96 http://demo08.herokuapp.com/sd

File: views/404.jade [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  .errorDiv
+    .error-code.m-b-10.m-t-20
+      | 404 
+      i.fa.fa-warning
+    h3.font-bold We couldn't find the page..
+    .error-desc
+      | Sorry, but the page you are looking for was either not found or does not exist. 
+      div
+        a.login-detail-panel-button.btn(href='/')
+          i.fa.fa-arrow-left
```
Comment (imujjwal96) [inline views/404.jade]: is this favicon rendering?

Comment (sch00lb0y) [inline views/404.jade]: I forgot to add the font awesome css. I'll add now. Thanks

Comment (imujjwal96) [inline views/404.jade]: :+1:

Comment (sch00lb0y) [inline views/404.jade]: I have updated. please check

**Comment by sch00lb0y** [comment]
@pri22296 @imujjwal96 please check this

**Comment by imujjwal96** [comment]
@sch00lb0y Was there any update?

**Comment by sch00lb0y** [comment]
after rebase, your review got dismissed.

**Pull request merged**